Lib/test/test_strptime.py

```diff
@@ -299,6 +330,28 @@ def helper(self, directive, position):
                          (directive, strf_output, strp_output[position],
                           self.time_tuple[position]))
 
+    def helper_for_directives_accepting_few_digits_year(self, directive):
+        fmt = "%" + directive
+
+        try:
+            (input_string,
+             expected_year,
+             ) = _get_data_to_test_strptime_with_few_digits_year(fmt)
+        except AssertionError as exc:
+            self.fail(str(exc))
+
+        try:
+            output_year = _strptime._strptime(input_string, fmt)[0][0]
```

Comment:
Is't possible for `_strptime._strptime` to return a one-dim list?

Reply:
It *always* returns a 3-`tuple`, the 0th item of which is a 9-`tuple`, the 0th item of which is a `int` representing the year number.